packages/rs-drive/src/drive/protocol_upgrade/mod.rs

```diff
@@ -308,4 +366,174 @@ impl Drive {
 
         Ok(value_changed)
     }
+
+    /// Removes the validator proposed app version
+    /// returns true if the value was removed
+    /// returns false if it never existed
+    pub(crate) fn remove_validator_proposed_app_version_operations(
```

Comment:
I've re-read this several times and I still can quite figure out what effect this should have. We have an allowed list of versions the proposer can propose, they're stored in Grove, and this function sets new valid versions?